### ISSUE TITLE
BUG: Static Custom Anomalies fail to upload

### ISSUE THREAD

**Issue by colinsidi** [dynatrace-oss/terraform-provider-dynatrace#42]
Sample in below config is over-written with "0" duration terraform apply

using tf file containing:
```
resource "dynatrace_custom_anomalies" "sample_name" {
  name = "sample_name"
  description = "some description"
  enabled = true
  aggregation_type = "AVG"
  disabled_reason = "NONE"
  metric_id = "builtin:host.disk.usedPct"
  severity = "CUSTOM_ALERT"
  warning_reason = "NONE"
  dimensions {
    entity {
      key = "dt.entity.disk"
      filter {
        operator = "EQUALS"
        value = "/local"
      }
    }
  }
  scopes {
    management_zone {
      id = "-189204438944455158"
    }
  }
  strategy {
    static {
      alert_condition = "ABOVE"
      alerting_on_missing_data = false
      dealerting_samples = 5
      **samples = 5**
      threshold = 97
      unit = "PERCENT"
      violating_samples = 3
    }
  }
}
```
Request send is:
```
2021/06/03 15:52:32   Request Body: {"aggregationType":"VALUE","alertingScope":[{"filterType":"MANAGEMENT_ZONE","mzId":"8204167837028444715"}],"description":"The {metricname} value of {severity} was {alert_condition} your custom threshold of {threshold}.","disabledReason":"NONE","enabled":true,"metricId":"builtin:service.errors.fivexx.count","monitoringStrategy":{"alertCondition":"ABOVE","dealertingSamples":5,"**samples":0,"**threshold":10,"type":"STATIC_THRESHOLD","unit":"COUNT","violatingSamples":3},"name":"KS QA 5xx Increased terraform2","severity":"ERROR","warningReason":"NONE"}
2021/06/03 15:52:33   400 Bad Request
```
Error from terrafrom:
```
Error: {
  "code": 400,
  "message": "Constraints violated.",
  "constraintViolations": [
    {
      "parameterLocation": "PAYLOAD_BODY",
      "location": "",
      "message": "must be greater than or equal to 3",
      "path": "monitoringStrategy.samples"
    },
    {
      "parameterLocation": "PAYLOAD_BODY",
      "location": "",
      "message": "The 'violatingSamples' number must be in between 1 and the given 'samples' number.",
      "path": "monitoringStrategy.violatingSamples"
    }
  ]
}
```

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Bug confirmed. It will be fixed with v1.2.2 - likely to be release by tomorrow. I will keep this issue open until then.

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hello @colinsidi 
Release [v1.2.2](https://github.com/dynatrace-oss/terraform-provider-dynatrace/releases/tag/v1.2.2) serializes the attribute `samples` properly now. Verified with the exact example you've provided.
